__tests__/app.test.js

```diff
@@ -49,5 +71,13 @@ describe("GET", ()=>{
                     expect(body).toEqual({status: 404, msg: "Not Found"})
                 })
         })
+        it("should return status 400: Bad Request when given an invalid path", ()=>{
+            return request(app)
+                .get('/api/reviews/not-a-review-id')
+                .expect(400)
+                .then(({body})=>{
+                    expect(body).toEqual({status: 400, msg: 'bad request'})
+                })
+        })
```

Comment:
What about if we requested an id that is out of range like 10000, what kind of response would we expect?